kfdef/README.md

```diff
@@ -0,0 +1,33 @@
+# Deploy latest DSPO via ODH 
+
+To deploy the latest DSPO using the changes within this repo via Open Data Hub you can follow these steps
+
+
+## Pre-requisites
+1. An OpenShift cluster that is 4.10 or higher.
+2. You will need to be logged into this cluster as [cluster admin] via [oc client].
+3. The OpenShift Cluster must have OpenShift Pipelines 1.9 or higher installed. Instructions [here][OCP Pipelines Operator].
+4. The Open Data Hub operator needs to be installed. You can install it via [OperatorHub][installodh].
+
+
+## Deploy Kfdef
+
+Clone this repository then run the following commands: 
+
+```bash 
+# If this namespace does not exist
+oc new-project odh-applications 
+
+# Then run
+oc apply -f https://raw.githubusercontent.com/opendatahub-io/data-science-pipelines-operator/main/kfdef/kfdef.yaml -n odh-applications
```

Comment:
I'm testing the kfdef install, is this url incorrect? I see a 404.

`error: unable to read URL "https://raw.githubusercontent.com/opendatahub-io/data-science-pipelines-operator/main/kfdef/kfdef.yaml", server reported 404 Not Found, status code=404`